pkg/requirements/requirements.go

```diff
@@ -64,23 +64,9 @@ func GetRequirementsAndGit(g gitclient.Interface, gitURL string) (*jxcore.Requir
 func CloneClusterRepo(g gitclient.Interface, gitURL string) (string, error) {
 	// if we have a kubernetes secret with git auth mounted to the filesystem when running in cluster
 	// we need to turn it into a git credentials file see https://git-scm.com/docs/git-credential-store
-	secretMountPath := os.Getenv(credentialhelper.GIT_SECRET_MOUNT_PATH)
-	if secretMountPath != "" {
-		err := credentialhelper.WriteGitCredentialFromSecretMount()
-		if err != nil {
-			return "", fmt.Errorf("failed to write git credentials file for secret %s : %w", secretMountPath, err)
-		}
-
-		gitURL, err = AddUserPasswordToURLFromDir(gitURL, secretMountPath)
-		if err != nil {
-			return "", fmt.Errorf("failed to add username and password to git URL: %w", err)
-		}
-	} else {
-		if kube.IsInCluster() {
-			log.Logger().Warnf("no $GIT_SECRET_MOUNT_PATH environment variable set")
-		} else {
-			log.Logger().Debugf("no $GIT_SECRET_MOUNT_PATH environment variable set")
-		}
+	gitURL, err := GitCredsFromCluster(gitURL)
```

Comment:
This reference will now be wrong. What IDE are you using?

Reply:
Ah, using Goland but didn't do the obvious replacement checks